tests/ignite/metrics/test_cosine_similarity.py

```diff
@@ -11,8 +11,8 @@
 from ignite.metrics import CosineSimilarity
 
 
-def test_zero_sample():
-    cos_sim = CosineSimilarity()
+def test_zero_sample(available_device):
```

Comment:
No need to add available_device fixture to the tests checking if an error is raised